components/MediaCaptionImage/MediaCaptionImage.tsx

```diff
@@ -99,9 +96,9 @@ const MediaCaptionImage = ({
   ...props
 }: MediaCaptionImageProps) => {
   return (
-    <WidthBox
+    <Container
       {...props}
-      boundingWidth={boundingWidth}
+      width="full"
```

Comment:
It doesn't seem like there is a reason to use `WidthBox` as the wrapper since the `WidthBox` in this repo doesn't actually have the `width` prop which allows it to span less than 12 of 12 columns. I actually removed `WidthBox` entirely since `MediaCaptionImage` is the only component that uses it.